hrc_speech_prediction/tests/test_context_model.py

```diff
@@ -7,20 +7,60 @@
 class TestNode(TestCase):
 
     def setUp(self):
+        self.empty_context = []
+        self.context = ['a','b','c']
+        self.contexts = [['a','b','c'],['d','e','f'],['g','h','i'],['a','g','e']]
         pass
 
     def test_empty_has_zero_n_children(self):
-        raise NotImplementedError
+        self.x = Node()
```

Comment:
Here you can use a local variable instead of creating an attribute of the TestCase since the node will only be used locally.
```python
x = Node()
self.assertEqual(x.n_children, 0)
```